### DISCUSSION TITLE
Cours OpenClassrooms - Démarrez votre projet avec java

### DISCUSSION THREAD

**Discussion by Frank2018-coder** [community/community#67641] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

onjour Mr Anthony, je suis étudiant sur OpenClassrooms parcours Java et présentement j'essaie de comprendre votre cours intitulé : "Démarrez votre votre projet Java" sauf que le repository Git que vous avez donnez dans le cours ne fonctionne pas.. comment faire? et je trouve votre cours très très intérsant.  [https://github.com/cursan-a/MyMenu-I.git](https://github.com/cursan-a/MyMenu-I.git?fbclid=IwAR0FMtWxnt-xMk2CWhM_1aDbkBDGKSA4gzJjG6z1OX7pqSnY7Wtz61fHrtY)

**Comment by selin-topcu** [comment]
This course may have been deleted or its visibility may have been changed to private

**Reply by mkos11** [reply]
hello people

**Comment by sahandmohammadrehzaii** [comment]
hello people

**Comment by Frank2018-coder** [comment]
Bonjour, dans le souci d'obtenir une licence gratuite pour l'utilisation de
IntelliJ idea en tant qu'étudiant j'ai rempli un formulaire sur leur plate
forme en inscrivant l'adresse mail de openclassrooms suivante: mailto:
***@***.*** veuillez me transmettre le message qu'ils vous
ont envoyé
[image: image.png]

**Reply by mkos11** [reply]
hello people

**Comment by qil50** [comment]
response #1

**Reply by qil500** [reply]
good

**Comment by qil50** [comment]
response #2

**Comment by mkos11** [comment]
hello people
